scenarios/arcaflow/memory-hog/input.yaml

```diff
@@ -0,0 +1,5 @@
+kubeconfig: {}
+node_selector: {}
+vm_bytes: 10%
+vm_workers: 2
+timeout: 30
```

Comment:
Thoughts on renaming the timeout to something along the lines of run_time?

Reply:
is duration ok?